__tests__/__snapshots__/test.js.snap

```diff
@@ -22933,12 +22934,13 @@ Array [
             "static": Array [],
           },
           "name": "getUndefined",
-          "namespace": "MyClass.getUndefined",
+          "namespace": ".MyClass.getUndefined",
```

Comment:
Looks like this is a regression? Namespaces shouldn't start with `.`.

Reply:
This is a strange case. In the tests `MyClass` is a member of `com.Test`. However  `com.Test` is never defined. Should the name space be `com.Test.MyClass.getUndefined`? or should it remain `MyClass.getUndefined`?

Reply:
@tmcw

Reply:
This should probably remain as `MyClass.getUndefined`, and the way we format namespaces should avoid prefixing static scopes with `.`

Reply:
@tmcw there's currently a [namespace](https://github.com/documentationjs/documentation/blob/master/__tests__/__snapshots__/test.js.snap#L39391)  on master in the test snapshots that starts with a `.`.

This is the same scenario as above except in this case the static is implied from the syntax tree instead of being assumed based on the `@memberof` tag.

Adding `memo &&` [here](https://github.com/documentationjs/documentation/blob/06ed2acb26924e5b1153512d71c16a80450e7922/src/hierarchy.js#L221) would remove the leading `.` in all cases including cases currently in master. Let me know how I should proceed.